### ISSUE TITLE
Diferenciate better "Settings" and "Preferences"

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#3145]
There are two sets of parameters: 
* "Settings" is a menu for application-wide user-configurable settings exposed in the library window, 
* "Preferences" is a publication-specific options panel in the reader window(s).

This may not be very clear and requires context, which is not always evident, especially for non-visual readers. 

Proposal: 

Add context to those button titles, like "Application settings" and "Reading preferences"

**Comment by arthur-lemeur** [comment]
In french, "settings" is translated as "préférences", should be "réglages".

**Comment by arthur-lemeur** [comment]
The issue with renaming "preferences" as "reading preferences" is that it would be split on 2 lines, especially in German.

**Comment by gautierchomel** [comment]
Maybe we can partially act by changing only the "settings" from the reading view to "Reading preferences" as it is a tooltip and therefore won't split in two lines.

**Comment by gautierchomel** [comment]
Also maybe add an Aria label to make context understanding better for "Applications settings"
